bin/check-openpmix.py

```diff
@@ -81,11 +82,12 @@ def check_missing_openpmix(std_all_refs, openpmix_all_refs,
             p.wait()
             if p.returncode != 0 and p.returncode != 1:
                 print("Error: Failed to grep and confirm missing item \""+std_ref+"\". grep error code "+str(p.returncode)+")");
-                sys.exit(2)
+                continue
```

Comment:
Don't we still want this to error out? This indicates that something is wrong with the script so the results are suspect.

Reply:
I wasn't entirely clear as to the reason for the error here. If it truly is something wrong with the script, then yes, we should error out.